edward/inferences.py

```diff
@@ -199,12 +199,10 @@ def build_score_loss(self):
         ELBO = E_{q(z; lambda)} [ log p(x, z) - log q(z; lambda) ]
         """
         x = self.data.sample(self.n_data)
-        z, self.samples = self.variational.sample(self.n_minibatch)
-
-        q_log_prob = tf.zeros([self.n_minibatch], dtype=tf.float32)
-        for i in range(self.variational.num_factors):
-            q_log_prob += self.variational.log_prob_i(i, tf.stop_gradient(z))
+        self.samples = self.variational.sample(self.n_minibatch)
+        z = self.samples
```

Comment:
I'm also keeping the statement `z = self.zs` so that the rest of the code is clean in applying functions to `x` and `z`. We can think about whether to not do this statement later.